trunk/SUAVE/Analyses/Aerodynamics/AVL_Inviscid.py

```diff
@@ -330,41 +228,30 @@ def build_surrogate(self):
 # ----------------------------------------------------------------------
         
     def evaluate_conditions(self,run_conditions):
-        """Process vehicle to setup geometry, condititon, and configuration.
-
-        Assumptions:
-        None
-
-        Source:
-        N/A
-
-        Inputs:
-        run_conditions <SUAVE data type> aerodynamic conditions; until input
-                method is finalized, will assume mass_properties are always as 
+        """ process vehicle to setup geometry, condititon and configuration
```

Comment:
What happened to the rest of the docstring?

Reply:
Issue resolved. Similar issue before of importing old scripts